packages/view-types/scatterplot-gating/src/GatingSubscriber.js

```diff
@@ -317,12 +319,14 @@ export function GatingSubscriber(props) {
   const cellRadius = (cellRadiusMode === 'manual' ? cellRadiusFixed : dynamicCellRadius);
   const cellOpacity = (cellOpacityMode === 'manual' ? cellOpacityFixed : dynamicCellOpacity);
 
+  const [uint8ExpressionData] = useUint8ExpressionData(expressionDataColor);
```

Comment:
Sorry, just reviewing, but shouldn't this have a second returned item, the extent?

Reply:
Yes but that is only needed for the legend, and this gating scatterplot does not have a legend (at least currently)